packages/messaging/src/controllers/window-controller.ts

```diff
@@ -318,7 +318,6 @@ export class WindowController extends BaseController {
               message_time: data[FN_CAMPAIGN_TIME],
               message_device_time: Math.floor(Date.now() / 1000)
             }
-            /* eslint-enable camelcase */
```

Comment:
But why? :(

Reply:
Argh, that's a bad merge, I keep putting it back in after merges/rebases but it keeps removing it again. I'll put it back.